newton/tests/kamino/test_kamino_solver_kamino.py

```
@@ -244,16 +264,16 @@ def assert_states_close_masked(
             )
         for attr in cts_attributes:
             np.testing.assert_allclose(
-                getattr(state, attr).numpy()[cts_offset[wid] : cts_offset[wid + 1]],
-                getattr(state_ref, attr).numpy()[cts_offset[wid] : cts_offset[wid + 1]],
+                getattr(state, attr).numpy()[offset[wid] : offset[wid + 1]],
```

Ruff (0.16.1)

[error] Undefined name `offset`

(F821)